scripts/services/migrations.ts

```diff
@@ -0,0 +1,24 @@
+import { compat, types as T } from "../deps.ts";
+
+export const migration: T.ExpectedExports.migration = compat.migrations
+  .fromMapping(
+    {
+      "1.4.4": {
+        up: compat.migrations.updateConfig(
+          (config) => {
+            return config;
+          },
+          true,
+          { version: "1.4.4", type: "up" },
+        ),
+        down: compat.migrations.updateConfig(
+          (config) => {
+            return config;
+          },
+          true,
+          { version: "1.4.4", type: "down" },
+        ),
+      },
+    },
+    "1.4.4",
+  );
```

Comment:
```suggestion
  );

```

Also, this migration doesn't appear to do anything?

Also, this is a self-migration?

Reply:
self-migration, made it easier to test. also doesn't do anything from other versions. any issue with that?

Reply:
I thought migrations that aren't defined are now `configured: true` @dr-bonez ?